ha/util/conf_store.py

```diff
@@ -221,3 +221,30 @@ def get_disk_list_for_cvg(index, node_id, cvg_id):
         except Exception as e:
             Log.error(f"Unable to fetch Disk list from GConf. Error {e}")
             raise Exception(f"Unable to fetch Disk list. Error {e}")
+
+    @staticmethod
+    def get_control_pods(index):
```

Comment:
suggestion: I think it is better to have the function name "get_control_nodes" but if it is aligned to the already written functions for server and data node ids then ok.
but we are referring to pod as a node everywhere and you are returning node ids.
Note: @gargpalak048 @mariyappanp

Reply:
@mukhtar-inamdar Agree, 
But  the func name aligns with the present func names.
Let's keep it this way only for now in my opinion.

Reply:
ok